webpages/settings/data/tags.js

```diff
@@ -44,6 +44,14 @@ export default [
       theme: true,
     },
   },
+  {
+    name: "forPlayer",
+    matchName: "player",
+    color: "darkgreen", // TODO: use a new tag color
```

Comment:
reminder

Reply:
I'll do that later today or tomorrow.

Reply:
I think it's been decided there'll be no theme sub-category, so I'll just remove the tag completely.

Reply:
I re-added the theme category, but now #4989 is open and if it gets merged, the colour of all theme tags will be the same.